doc/releases/pacific.rst

```diff
@@ -2,6 +2,240 @@
 Pacific
 =======
 
+v16.2.15 Pacific
+================
+
+This is the fifteenth backport release in the Pacific series.
```

Comment:
Need to mention that it's expected to be the last backport release in the series here.

Reply:
Didn't we say that about 16.2.14?

"You said that two Dunes ago!"